bridge/qb/client/functions.lua

```diff
@@ -86,56 +121,97 @@ end
 
 -- Getters
 
----@deprecated use GetVehicles from imports/utils.lua
-functions.GetVehicles = GetVehicles
+---@deprecated use the GetGamePool('CVehicle') native directly
+functions.GetVehicles = function()
+    GetGamePool('CVehicle')
+end
 
----@deprecated use GetObjects from imports/utils.lua
-functions.GetObjects = GetObjects
+---@deprecated use the GetGamePool('CObject') native directly
+functions.GetObjects = function()
+    GetGamePool('CObject')
+end
 
----@deprecated use GetPlayersInScope from imports/utils.lua
+---@deprecated use the GetPlayersInScope native directly
 functions.GetPlayers = GetPlayersInScope
 
----@deprecated use GetPeds from imports/utils.lua
-functions.GetPeds = GetPeds
+---@deprecated use the GetGamePool('CPed') native directly
+functions.GetPeds = function()
```

Comment:
The original [`QBCore.Functions.GetPeds`](https://github.com/qbcore-framework/qb-core/blob/cdd1502f0157fe2c9e1d97ef1b6e79eceb9e6c9f/client/functions.lua#L169) actually supports an `ignoreList` that was implemented in utils. We should probably re-implement this inside the bridge.